app/assets/javascripts/modules/moj.matchVisitors.js

```diff
@@ -211,6 +218,10 @@
       return isBanned;
     },
 
+    isContactRestricted: function(contact) {
+      return contact && contact.restrictions && contact.restrictions.length > 0? true : false;
```

Comment:
@chrishanes - if / else statement  not required here? ie just return bool statement

Reply:
What do you mean @TobyRet ?

Reply:
I think you could drop the `? true : false` as what's before will return true or false

Reply:
i think...

Reply:
Ah, OK